src/main/java/io/logz/logback/LogzioLogbackAppender.java

```diff
@@ -220,7 +217,15 @@ private String getValueFromSystemEnvironmentIfNeeded(String value) {
     }
 
     private JsonObject formatMessageAsJson(ILoggingEvent loggingEvent) {
-        JsonObject logMessage = new JsonObject();
+        JsonObject logMessage;
+
+        try {
+            JsonElement jsonElement = gson.fromJson(loggingEvent.getFormattedMessage(), JsonElement.class);
+            logMessage = jsonElement.getAsJsonObject();
+        } catch (Exception e) {
```

Comment:
There will be performance penalties for someone who is not using json format messages. Can you add to the constructor a `format` variable with two options json or text and the default will be text. Then use if condition to minimize the performance penalties for those who use text format